### ISSUE TITLE
CCQuadParticleSystem - How to use ?

### ISSUE THREAD

**Issue by AmitNHB** [ZhouWeikuan/cocos2d#69]
Hello, I'm sorry if this is a total newbie question but I can't figure out how to use the CCQuadParticleSystem class. I have my .plist file in my asset folder - the same file which I used with Cocos2D-X and is working just fine. I tried both:

CCQuadParticleSystem p1 = (CCQuadParticleSystem)CCQuadParticleSystem.particleWithFile("fire.plist");

and

CCParticleSystem p2 = CCQuadParticleSystem.particleWithFile("fire.plist");

and both times I get null.

I tried searching for examples and tutorials but just can't figure this one thing out. Everything else seems to be working great. Thanks a lot.

If this is of any used, I uploaded the .plist file I'm trying to work with here:
http://www.megafileupload.com/en/file/396356/fire-plist.html

**Comment by clin407** [comment]
Hey Amit, did you ever figure out this issue?

**Comment by AmitNHB** [comment]
No, sorry. Honestly. That whole project sucked. Use http://www.cocos2d-x.org
if you're looking for a free solution or http://unity3d.com if you're
willing to pay some money for a quality engine.

Good luck !
